cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -196,6 +196,7 @@
     rebalance.add(EXCLUDE_RECENTLY_DEMOTED_BROKERS_PARAM);
     rebalance.add(EXCLUDE_RECENTLY_REMOVED_BROKERS_PARAM);
     rebalance.add(REPLICA_MOVEMENT_STRATEGIES_PARAM);
+    rebalance.add(IGNORE_PROPOSAL_CACHE_PARAM);
```

Comment:
also need to add this new parameter to `proposals` Map?

Reply:
It is already in the `proposals` set (not map), am I missing something?

Reply:
Ah, my bad. I just check the diff and it it is added to `rebalance` not to `proposal`, forget it is not a new parameter.